docs/source/cpp/tutorials/basic_arrow.rst

```diff
@@ -0,0 +1,284 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+.. or more contributor license agreements.  See the NOTICE file
+.. distributed with this work for additional information
+.. regarding copyright ownership.  The ASF licenses this file
+.. to you under the Apache License, Version 2.0 (the
+.. "License"); you may not use this file except in compliance
+.. with the License.  You may obtain a copy of the License at
+
+..   http://www.apache.org/licenses/LICENSE-2.0
+
+.. Unless required by applicable law or agreed to in writing,
+.. software distributed under the License is distributed on an
+.. "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+.. KIND, either express or implied.  See the License for the
+.. specific language governing permissions and limitations
+.. under the License.
+
+.. default-domain:: cpp
+.. highlight:: cpp
+
+.. cpp:namespace:: arrow
+
+===========================
+Basic Arrow Data Structures
+===========================
+
+Apache Arrow provides fundamental data structures for representing data:
+:class:`Array`, :class:`ChunkedArray`, :class:`RecordBatch`, and :class:`Table`. 
+This article shows how to construct these data structures from primitive 
+data types; specifically, we will work with integers of varying size 
+representing days, months, and years. We will use them to create the following data structures:
+
+#. Arrow :class:`Arrays <Array>`
+#. :class:`ChunkedArrays<ChunkedArray>` 
+#. :class:`RecordBatch`, from :class:`Arrays <Array>`
+#. :class:`Table`, from :class:`ChunkedArrays<ChunkedArray>` 
+
+Pre-requisites
+--------------
+Before continuing, make sure you have:
+
+#. An Arrow installation
```

Comment:
Do we want to link to the doc page?

Reply:
Sure! Easy enough link to use a few times.